flw/src/main/java/org/motechproject/nms/flw/utils/FlwConstants.java

```diff
@@ -22,6 +22,7 @@
     public static final String NON_CENSUS_VILLAGE_ID = "SVID";
     public static final String TYPE = "Type";
     public static final String UPDATED_ON = "Updated_On";
+    public static final String VALID_TYPE = "ASHA";
```

Comment:
non-blocking comment - but this is not potentially always true right? Valid type could be extended to asha & dosha tomorrow. and then you will have to check for designation.equals asha || dosha, etc. 

Just call it ASHA_TYPE in the future.